wrapping/test/PythonANTSRegistrationTest.py

```diff
@@ -0,0 +1,58 @@
+#==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          https://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#==========================================================================*/
+
+import itk
+import argparse
+
+parser = argparse.ArgumentParser(description="Register moving image to fixed image.")
+parser.add_argument("-f", "--fixed-image", required=True)
+parser.add_argument("-m", "--moving-image", required=True)
+parser.add_argument("-i", "--initial-transform", required=False)
+parser.add_argument("-o", "--output-transform", required=True)
+parser.add_argument("-r", "--resampled-moving", required=False)
+args = parser.parse_args()
+
+itk.auto_progress(2)
+
+fixed_image = itk.imread(args.fixed_image)
+moving_image = itk.imread(args.moving_image)
+
+PixelType = itk.template(fixed_image)[1][0]
+Dimension = fixed_image.GetImageDimension()
+ImageType = itk.Image[PixelType, Dimension]
+assert ImageType == type(fixed_image)
+
+# registration_method = itk.ANTSRegistration[ImageType, ImageType].New(fixed_image, moving_image)
```

Comment:
Uncommenting this line causes:
```log
Loading ANTsWasm... done
Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1551, in set_inputs
    new_itk_object.SetInput(setInputNb, arg)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'itkANTSRegistrationDISS3' object has no attribute 'SetInput'. Did you mean: 'GetInputs'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1570, in set_inputs
    raise TypeError("Object accepts only 1 input.")
TypeError: Object accepts only 1 input.
python-BaseException

Process finished with exit code 1
```
and so does attempt to use it via procedural interface:
```py
forward_transform, inverse_transform = itk.ants_registration(
    fixed_image, moving_image, initial_transform=initial_transform)
```